app/tests/test_admin_page.py

```diff
@@ -606,60 +606,6 @@ def setUp(self):
     def test_add_asset_minimal(self):
         self._create_asset_minimal(self.item)
 
-    @mock_s3_asset_file
-    def test_add_update_asset(self):
-
-        asset, data = self._create_asset(self.item)
-
-        filecontent = b'mybinarydata2'
-        filelike = BytesIO(filecontent)
-        filelike.name = 'testname.tiff'
-
-        # update some data
-        data["description"] = "This is a new description"
-        data["eo_gsd"] = 20
-        data["geoadmin_lang"] = "fr"
-        data["geoadmin_variant"] = "kombs"
-        data["proj_epsg"] = 2057
-        data["title"] = "New Asset for test"
-        data["media_type"] = "application/x.ascii-grid+zip"
-        data["file"] = filelike
-        response = self.client.post(reverse('admin:stac_api_asset_change', args=[asset.id]), data)
-
-        # Status code for successful creation is 302, since in the admin UI
-        # you're redirected to the list view after successful creation
-        self.assertEqual(response.status_code, 302, msg="Admin page failed to update asset")
-        asset.refresh_from_db()
-        for key, value in data.items():
-            if key in ['item', 'id', 'file']:
-                continue
-            self.assertEqual(getattr(asset, key), value, msg=f"Failed to update field {key}")
-
-        # Check that file content has changed
-        with asset.file.open() as fd:
-            self.assertEqual(filecontent, fd.read())
-
-    def test_rename_asset(self):
-
-        asset, data = self._create_asset(self.item)
-
-        data['name'] = 'new_asset_name.zip'
-        # We just update the name hence we have to remove the
-        # 'file' from the data since submitting an empty file
-        # is not allowed
-        data.pop('file')
-
-        response = self.client.post(reverse('admin:stac_api_asset_change', args=[asset.id]), data)
-        self.assertEqual(response.status_code, 302)
-
-        # Un-comment with BGDIINF_SB-1625
-        # Assert that the location on s3 has been changed
-        # new_path = f"{asset.item.collection.name}/{asset.item.name}/{data['name']}"
-        # self.assertS3ObjectExists(new_path)
-
-        # asset.refresh_from_db()
-        # self.assertEqual(asset.file.name, new_path)
```

Comment:
This test should also be left but changed to make sure that renaming is not allowed. If I remember well in case of failure the status code is 200 and 302 in case of success for admin page request.

Reply:
This still needs to be re-added and modified

Reply:
@rebert I guess you didn't find a way to easily test if the name field in admin page is read only ?